app/services/senders/message_sender.rb

```diff
@@ -1,18 +1,52 @@
 class MessageSender < AbstractSender
-
   # How to use:
+  # As a message:
+  # sender = MessageSender.new(recipient)
+  # sender.set_message('Hi mom')
+  #       .deliver!
+  #
+  # As a quick reply:
```

Comment:
Can we change this to 'To send a message with quick replies?' What do you think?